### DISCUSSION TITLE
AnyCubic Photon Mono X - PrusaSlicer doesn't recognize bottom layer or normal layer exposure time

### DISCUSSION THREAD

**Discussion by IdleStep** [sn4k3/UVtools#798] [Q&A] [answered]
I am trying to set the parameters in PrusaSlicer for Exposure time, it is not recognized in the "START_CUSTOM_VALUES" profile notes. Curious if anyone has tips or guidance to setting exposure time from the slicer instead of editing within UV tools?

Printer: AnyCubic Photon Mono X 

```
START_CUSTOM_VALUES
WaitTimeBeforeCure_2.5
BottomLiftHeight_8
LiftHeight_8
BottomLiftSpeed_120
LiftSpeed_120
RetractSpeed_120
BottomLightPWM_255
LightPWM_255
END_CUSTOM_VALUES
```

**Comment by sn4k3** [comment, marked as answer]
Custom values are only for configurations that are not present in slicer.
The parameters you seek are present in the "Material settings" tab.

![image](https://github.com/sn4k3/UVtools/assets/113281/0a0ff0ae-3f24-4e41-ad4c-5fecad887b06)

**Comment by IdleStep** [comment]
Thank you! I did not realize that the import was also looking at typical slicer settings. Makes sense, now that you've explained it! :)
